AzureIndexer/Stratis.Bitcoin.Features.AzureIndexer/Checkpoint.cs

```diff
@@ -97,10 +138,12 @@ public bool SaveProgress(BlockLocator locator)
 
         private async Task<bool> SaveProgressAsync()
         {
+            this.logger.LogTrace("()");
+
             var bytes = BlockLocator.ToBytes();
             try
             {
-
+                this.logger.LogTrace("Uploading block locator bytes");
                 await _Blob.UploadFromByteArrayAsync(bytes, 0, bytes.Length, new AccessCondition()
```

Comment:
Any mileage in logging the byte size?

Reply:
Not really. 
Block locator size depends only on the height of the block we are getting the locator from so approximately all locators are the same size